content/applications/general/iot/config/troubleshooting.rst

```diff
@@ -343,6 +343,153 @@ Save`.
+menu. Click into the printer in question, and select :guilabel:`Maintenance` in the first drop-down
+menu. Then, select :guilabel:`Modify Printer` in the second drop-down menu.
+
+.. image:: troubleshooting/main-modify.png
+   :align: center
+   :alt: Modify the make and model of the DYMO LabelWriter. Maintenance and Modify drop-down menus
+         highlighted.
+
+Next, select the specific network connection/printer that the modification should be made on. Click
+:guilabel:`Continue`.
+
+.. image:: troubleshooting/modify-select-printer.png
+   :align: center
+   :alt: Printer selection screen with Continue highlighted.
+
+On the next page, click :guilabel:`Continue` to proceed to set the :guilabel:`Make` of the printer.
+
+.. image:: troubleshooting/modify-printer-dymo.png
+   :align: center
+   :alt: Printer modification screen with Continue highlighted.
+
+Under :guilabel:`Make` select :guilabel:`DYMO` from the menu. Click on :guilabel:`Continue` to set
+the :guilabel:`Model`.
+
+.. image:: troubleshooting/setting-make.png
+   :align: center
+   :alt: Setting the make page, with DYMO and continue highlighted.
+
+On the following page, set the :guilabel:`Model` to :guilabel:`DYMO LabelWriter 450 DUO Label (en)`
+(or whichever DYMO printer model is being used). Click on :guilabel:`Modify Printer` to complete
+setting the new driver, a confirmation page will appear.
+
+.. image:: troubleshooting/setting-model.png
+   :align: center
+   :alt: Setting the printer model page with DYMO LabelWriter 450 DUO Label (en) highlighted.
+
+After being redirected to a confirmation page, acknowledging a successful update, click on the
+:guilabel:`Printers` button in the top menu.
+
+All the printers installed on the OpenPrinting CUPS server appear, including the newly updated:
+:guilabel:`DYMO LabelWriter 450 DUO Label` (or whichever DYMO printer model is being used). Click
+into the printer that was just updated.
+
+To print a test label, click on the :guilabel:`Maintenance` drop-down menu to the left of the
+:guilabel:`Administration` drop-down menu, and select :guilabel:`Print Test Page`. The test label
+will print out with a ten-second delay if the driver update was successful.
+
+.. image:: troubleshooting/print-test.png
+   :align: center
+   :alt: Printing a test page from the administration drop-down menu in the OpenPrinting CUPs
+         server.
+
+To reduce this delay a new printer will need to be added, follow the process below.
+
+DYMO LabelWriter print delay
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To resolve the delay issue after modifying the driver, the printer **must** be reinstalled. To
+reinstall the printer, open the OpenPrinting CUPS administration page by clicking
+:guilabel:`Printers server`, at the bottom of the :abbr:`IoT (Internet of Things)` box homepage.
+Then, click on :guilabel:`Administration` in the top menu, then click :guilabel:`Add a Printer`.
```

Comment:
Suggesting to use menuselection here since were directing the user through menus :)
```suggestion
::menuselection:`Printers server`, at the bottom of the :abbr:`IoT (Internet of Things)` box
homepage. Then, click on :menuselection:`Administration` in the top menu, then click :guilabel:`Add
a Printer`.
```